apps/condo/domains/billing/schema/RegisterBillingReceiptFileService.test.js

```diff
@@ -117,7 +117,7 @@ describe('RegisterBillingReceiptFileService', () => {
             const [dataCreated] = await registerBillingReceiptFileByTestClient(utils.clients.admin, payload)
             const [dataUpdated] = await registerBillingReceiptFileByTestClient(utils.clients.admin, payload)
             expect(dataCreated.id).toEqual(dataUpdated.id)
-            expect(dataUpdated.status).toEqual(REGISTER_BILLING_RECEIPT_FILE_SKIPPED_STATUS)
+            expect(dataUpdated.status).toEqual(REGISTER_BILLING_RECEIPT_FILE_UPDATED_STATUS)
```

Comment:
Why you change returned status type?

Reply:
I changed the logic of this service. Now it never returns skip status